src/modules/EctsSubject/Degree/DegreeOptions.ts

```diff
@@ -0,0 +1,10 @@
+export const DegreeOption = [
+  {
+    label: 'Studia I stopnia',
+    value: 'bachelor',
```

Comment:
value type probably should be an enum

Reply:
Imo https://bobbyhadz.com/blog/typescript-map-through-enum <- it looks more unclearly, so maybe better option is to make it `export const DegreeOption = [
  {
    label: 'Studia I stopnia',
    value: 'bachelor',
  },
  {
    label: 'Studia II stopnia',
    value: 'master',
  },
] as const;
` It will be more safe, but I don't have strong opinion here, so up to U